src/components/ContextData/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React,{ createContext, useContext, useState } from 'react'
+import data from '../../data'
+
+export const ContextData = createContext()
+
+export const DataProvider = ({ initalState = {}, children }) => {
+  const [values, setValues] = useState(initalState)
```

Comment:
Same as when it was in the App component, but now we can optionally set that initial state in `index.js`.